_pages/documentation/general_docs/gpu_models/gpufs.md

```diff
@@ -0,0 +1,158 @@
+
+```
+./build.sh
+```
+
+This process takes approximately 15-20 minutes and is mostly bound by download speed as a majority of the time is spent downloading Ubuntu packages.
+
+Building the disk image will also extract the Linux kernel. The extracted Linux kernel *must* be used with the disk image. In other words, you cannot input an arbitrary kernel to gem5 otherwise the GPU driver may not load successfully.
+
+After this process your environment should contain:
+* Disk image: `gem5-resources/src/x86-ubuntu-gpu-ml/disk-image/x86-ubuntu-gpu-ml`
+* Kernel: `gem5-resources/src/x86-ubuntu-gpu-ml/vmlinux-gpu-ml`
+
+#### Building GPU applications
+
+The GPU model is designed to run unmodified GPU binaries. If you have an application which runs on AMD GPU hardware and that hardware is supported in gem5, you can run the same binary in gem5. Note that as this is simulation, the application will need to be scaled down to a reasonable size to simulate in a realistic amount of time.
+
+Building applications for the GPU model is similar to [cross compiling](https://www.gem5.org/documentation/general_docs/compiling_workloads/) when the simulated ISA does not match the host. Either you must have the development tools installed locally or containerization like Docker can be used. Docker images to build GPU applications are provided with gem5 in [util/dockerfiles/gpu-fs](https://github.com/gem5/gem5/tree/stable/util/dockerfiles/gpu-fs). You may either build this image or use the gem5 provided image at `ghcr.io/gem5/gpu-fs`. The instruction below will use the gem5 provided docker.
+
+[Square](https://github.com/gem5/gem5-resources/tree/stable/src/gpu/square) is a simple application provided in gem5-resources which can be used to get started with the model. Generally, the `src/gpu` directory of gem5-resources contains a `Makefile.default` which is used to build a native application and `Makefile.gpufs` which contains application annotated with [m5ops](https://www.gem5.org/documentation/general_docs/m5ops/) that will only run within gem5.
+
+To build square using the gem5 provided docker image, navigate to the square directory and use the `Makefile.default` Makefile:
+
+```
+cd gem5-resources/src/gpu/square
+docker run --rm -u $UID:$GID -v $PWD:$PWD -w $PWD ghcr.io/gem5/gpu-fs make -f Makefile.default
+```
+
+The square binary should then be located at `gem5-resources/src/gpu/square/bin.default/square.default`
+
+#### Testing GPU application
+
+The GPU model provides multiple gfx9 configurations to simulate GPU applications. In the gem5 repository, these are:
+* MI300X: `configs/example/gpufs/mi300.py`
+* MI210 / MI250: `configs/example/gpufs/mi200.py`
```

Comment:
Do you think people will be confused why the file is named mi200.py but the documentation says it models MI210/250?

Reply:
Not sure. I don't really want to rename the files at this point. A better name would be gfx90a.py / gfx942.py but those names are meaningless to most people where as the marketing name (MI300X / MI250) is more well known.

I suppose technically the bios ROM used in mi200.py is an MI210 so maybe renaming to mi210.py is a decent compromise.

Reply:
Not sure. I don't really want to rename the files at this point. A better name would be gfx90a.py / gfx942.py but those names are meaningless to most people where as the marketing name (MI300X / MI250) is more well known.

I suppose technically the bios ROM used in mi200.py is an MI210 so maybe renaming to mi210.py is a decent compromise.

Reply:
I think it's fine to leave as is then, and see what folks ask.  I agree with your point.